docs/components/multiselect.md

```diff
@@ -315,55 +338,31 @@ See [autosuggest](#autosuggest) for more details.
   <KButton @click="clearIt">Clear</KButton>
 </ClientOnly>
 
-```html
-<div>
+```vue
+<template>
   <KLabel>Value:</KLabel> {{ myVal }}
   <KMultiselect v-model="myVal" :items="items" />
   <KButton @click="clearIt">Clear</KButton>
-</div>
-
-<script lang="ts">
-import { defineComponent } from 'vue'
+</template>
 
-export default defineComponent({
-  data() {
-    return {
-      myVal: ['cats', 'bunnies'],
-      items: [{
-        label: 'Cats',
-        value: 'cats',
-        selected: true
-      }, {
-        label: 'Dogs',
-        value: 'dogs',
-        selected: true,
-        disabled: true
-      }, {
-        label: 'Bunnies',
-        value: 'bunnies',
-        selected: true
-      },
-      {
-        label: 'Lions',
-        value: 'lions'
-      }, {
-        label: 'Tigers',
-        value: 'tigers'
-      }, {
-        label: 'Bears',
-        value: 'bears'
-      }, {
-        label: 'A long & truncated item',
-        value: 'long'
-      }]
-    }
-  },
-  methods: {
-    clearIt() {
-      this.myVal = []
-    }
-  }
-})
+<script setup lang="ts">
```

Comment:
I can't comment on the `autosuggest` props section, but can you update the items to have non-similar values so it's easier to tell what you're searching for?

Reply:
Like how? It is easy to search by numbers

Reply:
e.g. different text?

Reply:
Added _odd_ / _even_ words in front of item